src/main/java/me/contaria/seedqueue/gui/wall/SeedQueueWallScreen.java

```diff
@@ -454,6 +454,7 @@ public boolean keyPressed(int keyCode, int scanCode, int modifiers) {
 
         if (keyCode == GLFW.GLFW_KEY_ESCAPE && Screen.hasShiftDown()) {
             ModCompat.standardsettings$onWorldJoin();
```

Comment:
This call to `ModCompat.standardsettings#onWorldJoin` should be removed, the cache load serves as its replacement.